Traceback (most recent call last):
  File "PortListener.py", line 64, in <module>
    raise SystemError("Lock file /tmp/clv.pid exist")
SystemError: Lock file /tmp/clv.pid exist
